TeensySF/smoothsw.ini

```diff
@@ -1,16 +1,59 @@
-# SmoothSwing algorithm version, should be 1 or 2.
+# SmoothSwing algorithm version, you probably want to set this to 2.
 Version=2
-# Swing sensetivity, degrees of rotations per second required to reach full volume.
+
+# Degrees of rotations per second required to reach full volume.
+# Default is 450.0 (any value)
 SwingSensitivity=450
-# How many percent the hum sound will decrease as swing increases.
+
+# Smoothswing volume multiplier (defaults to 3x normal volume)
+# Default is 3.0 (value between 1 and 5)
+MaxSwingVolume=3.0
+
+# What percent the hum sound will decrease as swing increases
+# Default is 75.0 (value between 1 and 100)
 MaximumHumDucking=75
-# Non-linear swing response, higher values makes it more non-linear.
-SwingSharpness=1.5
-# Degrees / second needed to register as a swing.
+
+# Non-linear swing response (higher values make it more non-linear)
+# Values greater than 1 will result in the Smoothswing sound staying quieter 
+# at lower speeds and then ramping up quickly to full volume a higher speeds. 
+# Values less than 1 will result in the Smoothswing volume ramping up quickly 
+# at lower speeds and then staying there as you approach full speed. 
+# the default is 1.75 (any value)
+SwingSharpness=1.75
+
+# Degrees per second needed to register as a smoothswing.
+# Default is 20.0 (1 to 360)
 SwingStrengthThreshold=20
+
 # Length of first transition in degrees.
+# Default is 45.0 (1 to 360)
 Transition1Degrees=45
+
 # Length of second transition in degrees.
+# Default is 160.0 (1 to 360)
 Transition2Degrees=160.0
-# Swing volume multiplier defaults to 3x normal volume.
-MaxSwingVolume=3.0
\ No newline at end of file
+
+# If not zero, swngNNN.wav or swngNNN.wav will
+# be played when we reach this swing speed.
+# Unit is degrees per second, 450 is a reasonable value.
+AccentSwingSpeedThreshold=450
+
+# If not zero AND accent swings are on, this defines the threshold for when
+# a swing is considered a slash. Unit is degrees per second **per second**.
+# NOTE - While 260 is the default value, it is subjective.
+# Something like 100 might work better.
+# The higher the accent swing threshold, the higher the slash threshold will need to be.
+AccentSlashAccelerationThreshold=260.0
+
+# NOTES

```

Comment:
This doesn't seem like the right place to explain how smoothswing works. Let's limit this to only explanations about values in this file.

Might be good to have this information on the wiki and just put a link to that in this file instead.

Reply:
made appropriate for this font.